data/sql/updates/pending_db_world/rev_1585757753946835500.sql

```diff
@@ -0,0 +1,9 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1585757753946835500');
```

Comment:
@aleigood Fix SQL rev.
http://www.azerothcore.org/wiki/How-to-create-a-PR

Thanks @Kitzunu